test/policy_ordering.go

```diff
@@ -78,19 +96,21 @@ func PolicyOrdering(labels ...string) bool {
 				DoRootComplianceTest(policyWithDepName, policiesv1.NonCompliant)
 			})
 			It("Should become pending again when the initial policy is deleted", func() {
+				cleanEvent()
```

Comment:
@JustinKuli could you please provide your input on this?

I don't think we want to include this here.